I/ImageDistances/Compat.toml

```diff
@@ -5,8 +5,15 @@ julia = "0.6-0.7"
 Distances = "0.5-0.8"
 
 ["0.0.2-0.1"]
-Colors = "0.8.2-0.9"
 ProgressMeter = "0.5.2-0.9"
 
+["0.0.2-0.2"]
+Colors = "0.8.2-0.9"
+
 ["0.1"]
 julia = "0.7-1.3"
+
+["0.2"]
+Distances = "0.8"
+ProgressMeter = "0.9"
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.